src/main/java/reposense/system/CustomLogFormatter.java

```diff
@@ -13,22 +11,21 @@
  */
 public class CustomLogFormatter extends SimpleFormatter {
 
-    private static final DateFormat dateFormat = new SimpleDateFormat("dd/MM/yyyy hh:mm:ss");
+    private static final DateFormat dateFormat = new SimpleDateFormat("hh:mm:ss");
 
     @Override
     public synchronized String format(LogRecord record) {
         StringBuilder builder = new StringBuilder();
         builder.append(dateFormat.format(new Date(record.getMillis()))).append(" - ");
-        builder.append("[").append(record.getLevel()).append("] - ");
-        builder.append(formatMessage(record));
-        builder.append("\n");
 
         Throwable t = record.getThrown();
 
         if (t != null) {
-            StringWriter sw = new StringWriter();
-            t.printStackTrace(new PrintWriter(sw));
-            builder.append(sw);
+            builder.append("An error has occurred");
```

Comment:
This information is not helpful. Let's just show the message to the user in-regardless.
For the stacktrace, let's move to the log file output.

Reply:
Stacktrace is already visible in the log file output. This one is just to configure the format of console output